pandas/core/series.py

```diff
@@ -1637,6 +1637,45 @@ def to_string(
             +----+----------+"""
         ),
     )
+    @overload
```

Comment:
I think this is triggering the doc-tests:

The overloads need to be above the the doc-decorator:


```py
@overload 
def to_markdown

@overload 
def to_markdown

@overload 
def to_markdown

@doc 
@deprecate_nonkeyword_arguments
def to_markdown
```

Reply:
Yep! Just saw the error and your message xD 

Should be fixed now!